lib/Centreon/Object/Relation/Host/Template/Host.php

```diff
@@ -70,6 +70,30 @@ public function insert($fkey, $skey = null)
         $this->db->query($sql, array($fkey, $skey, $order));
     }
 
+    /**
+     * Delete host template / host relation and linked services
+     *
+     * @param int $fkey - host template
+     * @param int $skey - host
+     * @return void
+     */
+    public function delete($fkey, $skey = null): void
+    {
+        global $pearDB;
+        $this->db->beginTransaction();
+        try {
+            parent::delete($fkey, $skey);
+            $pearDB = $this->db;
+            $centreon = true; // Needed so we can include file below
```

Comment:
why do you remove the global $centreon in DB-FUNC if you need it ?

Reply:
You can see here :
https://github.com/centreon/centreon/blob/19.04.x/www/include/configuration/configObject/host/DB-Func.php#L36
that $centreon needs to be set, otherwise it will exit().
In the method itself - I removed it because it is not used anywhere.

Reply:
that's why I don't understand you want affect a boolean instead of this : 
![image](https://user-images.githubusercontent.com/34628915/63437890-b6249780-c42b-11e9-9ea6-10f3e0a13b8f.png)
why not getting the $centreon using global ?